reactioncommerce/reaction
# Fix inconsistent tag grid views (resolve 2213)

Resolves #2213 

When a product was in the revision stage we were adding it to the product cursor without checking whether it met the tag criteria. This just adds an extra check for when we are filtering by tag, to ensure if the product has that tag.

### Steps to test

1. Login as admin
1. Create a new tag on the nav
1. Create a new product
1. Add the tag you created to the new product (don't publish)
1. Observe that product shows up under the proper tag and not under any other
